task/07-yield-tasks.js

```diff
@@ -82,7 +99,17 @@ function* getFibonacciSequence() {
  *
  */
 function* depthTraversalTree(root) {
-    throw new Error('Not implemented');
+    let s = [];
+    s.push(root);
+    while (s.length > 0) {
+        let v = s.pop();
+        yield v;
+        if (v.hasOwnProperty('children')) {
+            for (let child of v.children.reverse()) {
+                s.push(child);
+            }
+        }
+    }
 }
```

Comment:
Please, use variable names to indicate their purpose. let s, v, q, etc - bad practice!
https://learn.javascript.ru/variables#name-things-right